datamol/descriptors/__init__.py

```diff
@@ -26,7 +27,12 @@
 from .descriptors import n_saturated_rings
 from .descriptors import n_aromatic_atoms
 from .descriptors import n_aromatic_atoms_proportion
+from .descriptors import refractivity
+from .descriptors import n_rigid_bonds
+from .descriptors import n_stereo_centers
+from .descriptors import n_charged_atoms
+from .descriptors import compute_ring_system
```

Comment:
Maybe move this to the scaffold section. It's technically not a descriptor.

Reply:
yes 100%. I made myself the same comment but then I got lazy xD